packages/js/src/components/portals/MetaboxPortal.js

```diff
@@ -24,6 +24,6 @@ export default function MetaboxPortal( { target, store, theme } ) {
 
 MetaboxPortal.propTypes = {
 	target: PropTypes.string.isRequired,
-	store: PropTypes.object,
-	theme: PropTypes.object,
+	store: PropTypes.object.isRequired,
+	theme: PropTypes.object.isRequired,
```

Comment:
I assume you checked that we always set them

Reply:
Yes, [classic](https://github.com/Yoast/wordpress-seo/blob/0e144b30d51cbe0d1ed4f3e98df2311b86a6ad6d/packages/js/src/helpers/classicEditor.js#L81)- and [block](https://github.com/Yoast/wordpress-seo/blob/b3ecdb20e7732f577818dc377c8f7c413b7d9837/packages/js/src/initializers/block-editor-integration.js#L162)-editor